src/ddmd/dstruct.d

```diff
@@ -696,12 +706,16 @@ extern (C++) class StructDeclaration : AggregateDeclaration
     }
 
     /***************************************
-     * Return true if struct is POD (Plain Old Data).
+     * Determine if struct is POD (Plain Old Data).
+     *
      * This is defined as:
-     *      not nested
-     *      no postblits, destructors, or assignment operators
-     *      no 'ref' fields or fields that are themselves non-POD
+     *      1. not nested
+     *      2. no postblits, destructors, or assignment operators
+     *      3. no 'ref' fields or fields that are themselves non-POD
```

Comment:
There's also the option to implement https://issues.dlang.org/show_bug.cgi?id=16112 ;)